apps/atomy-q/API/app/Services/QuoteIntake/NormalizationOverrideService.php

```
@@ -0,0 +1,314 @@
+            return [
+                'line' => $line,
+                'readiness' => $readiness,
+            ];
+        });
+    }
+
+    /**
+     * @param array<string, mixed> $rawData
+     * @return array<string, mixed>
+     */
+    private function providerProvenanceSnapshot(NormalizationSourceLine $line, array $rawData): ?array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$line'. (undefined)

(UnusedFormalParameter)